docs/conf.py

```diff
@@ -106,11 +100,12 @@
 # The version info for the project you're documenting, acts as replacement for
 # |version| and |release|, also used in various other places throughout the
 # built documents.
+# The following are default values, that are rewritten later in the script.
 #
 # The short X.Y version.
-version = ""  # Is set by calling `setup.py docs`
+version = os.getenv("READTHEDOCS_VERSION", "")
```

Comment:
Who is setting this environment variable? I think we should make sure that git tag versions = pypi distributions = the rtd versions.

Reply:
I would rather completely remove this part as we set `version` later using `__vesion__`. This only confuses people.

Reply:
So this env variable is set in all environments by Read The Docs.
The reason why I added it here is to have a fallback mechanism when publishing the docs, in the case there is a problems with the package.

Reply:
Ok, should we add a comment then that it's later potentially overwritten? Or remove it at this point and put it in the try-except block directly? It might be confusing to readers who read `version =` and think, okay so it's an environment variable but later it's overwritten.

Reply:
I agree with the confusion, in the latest commit to this PR I collapsed both sections of the code manipulating version/release into a single one. Hopefully it avoids confusion.